lib/roll.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/frames'
+
+class Roll
```

Comment:
As mentioned above, this class currently needs a lot of information about the rest of the game, suggesting that it isn't quite describing a single Roll. A single roll probably shouldn't care too much about the game it's been rolled in.  Can you think of a more accurately descriptive name for this class? 

If not, maybe you need to think a bit about what its responsibilities are. Does it make sense for them to be grouped together like this?